frontend/templates/cucumber/.github/workflows/browser.yml

```
@@ -0,0 +1,42 @@
+name: Run Browser Tests
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - ready_for_review
+      - synchronize
+
+jobs:
+  browser:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      packages: write
+
+    steps:
+      - name: Step 1 - Checkout main branch from GitHub
+        uses: actions/checkout@v3
+
+      - name: Set up Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: "18.x"
```

```suggestion
          node-version: 18.x
```
Consistent formatting (we don't have linting for yml files)

---

Good catch.

We did trial linting with `yaml` files - but our CloudFormation templates interacted unfavourably with the linting, so we put it into the "do that latter" pile.